brainscore_vision/nwb_integration/create_assembly.py

```diff
@@ -0,0 +1,203 @@
+import re, json
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from brainio.assemblies import NeuronRecordingAssembly
+from brainio.stimuli import StimulusSet
+from brainscore_vision import load_ceiling
+from brainscore_vision.metric_helpers.transformations import CrossValidation
+from pynwb.file import NWBFile
+from typing import Tuple
+
+
+def get_neuroids(nwb_file: NWBFile, subject: str) -> pd.DataFrame:
+    #-----------------------------------------------------------------------------------------------------------------------------
+    # Get electrode metadata from nwb file.
+    #-----------------------------------------------------------------------------------------------------------------------------
+    data_list = []  
+    for i in range(len(nwb_file.electrodes['location'])):
+        data_dict       = {}
+        location_item   = nwb_file.electrodes['location'][i]
+        group_item      = nwb_file.electrodes['group'][i] 
+        bank_item       = nwb_file.electrodes['group_name'][i] 
+        label_item      = nwb_file.electrodes['label'][i]
+        try: label_item = int(nwb_file.electrodes['label'][i].split('_')[0])
+        except: label_item = nwb_file.electrodes['label'][i]
+
+        location_match = re.search(r'\[(\d+).0, (\d+).0, (\d+).0\]', location_item)
+        if location_match:
+            data_dict['col'] = location_match.group(2)
+            data_dict['row'] = location_match.group(1)
+            data_dict['elec'] = location_match.group(3) 
+
+        serialnumer = group_item.description.split('Serialnumber: ')[-1]
+        data_dict['arr'] = serialnumer
+
+        group_match = re.search(r"\['(\w+)', '(\w+)', '(\w+)'\]", group_item.location)
+        if group_match:
+            data_dict['hemisphere']  = group_match.group(1)
```

Comment:
same here with perhaps adding an `else` in case `re` does not find a match

Reply:
Is this to grab the brain region? If so, then going to echo Mike's statement about `timebins` below and recommend we check that what we parse matches the user input